docs/book/src/cronjob-tutorial/testdata/project/config/crd/patches/validation_schema_cronjobs.yaml

```diff
@@ -0,0 +1,5 @@
+- op: remove
+  path: /spec/validation/openAPIV3Schema/properties/spec/properties/jobTemplate/properties/metadata/properties/creationTimestamp
+
+- op: remove
+  path: /spec/validation/openAPIV3Schema/properties/spec/properties/jobTemplate/properties/spec/properties/template/properties/metadata/properties/creationTimestamp
```

Comment:
createionTimestamp is really causing trouble for us.
These are temporary workaround.

Reply:
I can add a section about this if we decide this is the route we want to go ATM.

Reply:
Created https://github.com/kubernetes-sigs/kubebuilder/issues/861 to track